br/backup-and-restore-use-cases.md

```diff
@@ -100,9 +100,14 @@ Use the `br backup` command to back up the single table data `--db batchmark --t
 #### Backup prerequisites
 
 * [Preparation for backup](#preparation-for-backup)
-* Configure a high-performance SSD hard disk host as the NFS server to store data, and all BR nodes and TiKV nodes as NFS clients. Mount the same path (for example, `/br_data`) to the NFS server for NFS clients to access the server.
+* Configure a high-performance SSD hard disk host as the NFS server to store data, and all BR nodes, TiKV nodes, and TiFlash nodes as NFS clients. Mount the same path (for example, `/br_data`) to the NFS server for NFS clients to access the server.
 * The total transfer rate between the NFS server and all NFS clients must reach at least `the number of TiKV instances * 150MB/s`. Otherwise the network I/O might become the performance bottleneck.
 
+ **Note:**
+>
+> * During data backup, because only data in leader replicas are backed up, even if there is a TiFlash replica in the cluster，BR can complete backup without mounting TiFlash nodes.
```

Comment:
```suggestion
> * During data backup, because only the data of leader replicas are backed up, even if there is a TiFlash replica in the cluster, BR can complete the backup without mounting TiFlash nodes.
```